docs/demo/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "demo.tutorialkit.dev",
+  "version": "0.0.1",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "astro": "astro",
+    "build_with_check": "astro check && astro build",
+    "build": "astro build",
+    "dev": "astro dev",
+    "preview": "astro preview",
+    "start": "astro dev",
+    "postbuild": "cp _headers ./dist/"
+  },
+  "dependencies": {
+    "@tutorialkit/components-react": "0.0.1-alpha.22",
```

Comment:
I will switch to using the `workspace` packages instead. But first I need this https://github.com/stackblitz/tutorialkit/pull/84 to land.